docs/mddocs/Quickstart/ollama_quickstart.md

```diff
@@ -183,3 +183,14 @@ An example process of interacting with model with `ollama run example` looks lik
 <a href="https://llm-assets.readthedocs.io/en/latest/_images/ollama_gguf_demo_image.png" target="_blank">
   <img src="https://llm-assets.readthedocs.io/en/latest/_images/ollama_gguf_demo_image.png" width=100%; />
 </a>
+
+### Troubleshooting
+
+#### Why model is always loaded again after several minutes
+Ollama will unload model from gpu memory in every 5 minutes as default. For latest version of ollama, you could set OLLAMA_KEEP_ALIVE=-1 to keep the model loaded in memory. Reference issue: https://github.com/intel-analytics/ipex-llm/issues/11608
+
+#### Error when executing  `ollama serve`
+If you meet `llama runner process has terminated: exit status 0xc0000135`, please check if you are in the right oneapi env, or if you executed `source /opt/intel/oneapi/setvars.sh` on Linux

```

Comment:
If you meet `llama runner process has terminated: exit status 0xc0000135` when executing `ollama serve`, this is most likely caused by the lack of sycl dependency. Please check
1. if you have installed conda and if you are in the right conda environment which has pip installed oneapi dependencies on Windows,
2. if you have executed `source /opt/intel/oneapi/setvars.sh` on Linux